crates/cast/bin/cmd/wallet/mod.rs

```diff
@@ -131,9 +136,29 @@ pub enum WalletSubcommands {
         #[clap(flatten)]
         raw_wallet_options: RawWallet,
     },
-    /// List all the accounts in the keystore default directory
+    /// List accounts from keystore default directory, hardware ledger and AWS KMS.
     #[clap(visible_alias = "ls")]
-    List,
+    List {
+        /// List all the accounts in the keystore default directory.
+        #[clap(long, help_heading = "List local accounts")]
+        dir: bool,
```

Comment:
this could take Option<Option<PathBuf>> so you can provide custom dirs to check

Reply:
added in latest commit